prod-refractr.yml

```diff
@@ -10,6 +10,21 @@ default-headers:
   Strict-Transport-Security: '"max-age=60; includeSubDomains" always'
 
 refracts:
+  # OPST-305
+- dev.remote-settings.nonprod.webservices.mozgcp.net/:
+  - settings.dev.mozaws.net
+
+  # OPST-305
+- stage.remote-settings.nonprod.webservices.mozgcp.net/:
```

Comment:
What is the DSN of the origin then? (ie. without the CDN)

Reply:
GCP is quite different from AWS. On AWS, we had

    client -> CloudFront -> ELB (Origin) -> origin servers (ibidem)

On GCP, the load balancer is globally distributed, and the CDN is integrated into the load balancer. Remote Settings will run in pods in a Kubernetes cluster, and there will be no single IP address we could point a DNS record to to reach the origin.

It is possible to add another load balancer layer with the CDN disabled and a globally routable IP address, but I think it will be easier to leave the set-up as it currently is, and use cache-busting if we want to circumvent the CDN. Adding an additional layer adds complexity, cost and maintenance effort, and I'm not sure it's justified by the benefits we get out of that.

Reply:
Another option would be setting up some [cache bypass header](https://cloud.google.com/cdn/docs/caching#bypassing-cache), but I'm not sure how convenient that is in actual use.

Reply:
added a non-cdn reader endpoint per environment to the GCP LB [here](https://github.com/mozilla-it/webservices-infra/commit/4b296e7ed908b4acf081277be879a29ae703f0ac), updated the list of refracts :slightly_smiling_face: